src/main/kotlin/calculate/Calculate.kt

```diff
@@ -0,0 +1,57 @@
+package calculate
+
+import java.util.ArrayDeque
+import java.util.Deque
+import java.util.stream.Collectors
+
+class Calculate(private val validations: List<CalculateValidation>) {
+
+    constructor() : this(listOf(EmptyCheckValidation(), OperatorValidation()))
+
+    fun calculate(input: String): Int {
+        validateInput(input)
+        val operators = input.split("\\d+".toRegex()).filter { it.isNotBlank() }
+        val numbers: Deque<Int> = extractAndConvertNumber(input, operators)
```

Comment:
Kotlin 언어에서 제공하는 `타입추론` 기능을 적극적으로 활용해보면 어떨까요? 😊
```suggestion
        val numbers = extractAndConvertNumber(input, operators)
```

Reply:
몇개는 타입추론을 쓰고 몇개는 못썻네요.. 어서 익숙해 져야겠습니다 :)